swapfactory/utils.go

```diff
@@ -26,13 +26,13 @@ func GetSecretFromLog(log *ethtypes.Log, event string) (*mcrypto.PrivateSpendKey
 		return nil, errors.New("invalid event name, must be one of Claimed or Refunded")
 	}
 
-	abi, err := abi.JSON(strings.NewReader(SwapFactoryABI))
```

Comment:
I'm surprised the linter didn't flag a local variable named the same as an imported package.